Tiltfile

```diff
@@ -16,15 +18,15 @@ docker_build('tilt-site', 'src', dockerfile='site.dockerfile',
                run('bundle install', trigger=['src/Gemfile', 'src/Gemfile.lock'])
              ])
 
-docker_build('docs-site', '.', dockerfile='docs.dockerfile',
+docker_build('docs-site', '.', dockerfile='docs.dockerfile', ignore=["./api/api.py"],
```

Comment:
should this be in the other docker_build as well?

Reply:
nah, it doesn't fall within the build context of the other dockerfile.

Reply:
Ha actually, after merging master, yes it should!